pom.xml

```diff
@@ -227,6 +226,31 @@
     </pluginRepositories>
 
     <profiles>
+        <profile>
+            <id>openshift-it</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-failsafe-plugin</artifactId>
+                        <configuration>
+                            <systemPropertyVariables>
+                                <app.name>${project.artifactId}</app.name>
+                            </systemPropertyVariables>
+                            <classesDirectory>${project.build.directory}/${project.build.finalName}.${project.packaging}.original</classesDirectory>
+                        </configuration>
+                        <executions>
+                            <execution>
+                                <goals>
+                                    <goal>integration-test</goal>
+                                    <goal>verify</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
```

Comment:
What is the purpose of adding this profile and plugin?

Reply:
I was just following what had been done in https://github.com/snowdrop/spring-boot-http-booster/commit/1eac1c2 -- if we don't need it, I can definitely remove it and clean that up.

Reply:
Removed the extra profile

Reply:
as maven-failsafe-plugin wasn't used before seems that we effectively don't need it. I even prefer like that :-)

Reply:
Cool, so I will merge it.